lib/matchers.js

```diff
@@ -43,6 +43,15 @@ var polymerExternalStyle = p.AND(
   p.hasAttrValue('type', 'css')
 );
 
+var htmlImport = p.AND(
```

Comment:
we really need to consolidate these somewhere. I think some other projects don't have the `type` attribute check.